docs/integration-association.md

```diff
@@ -0,0 +1,37 @@
+# Association Integration
+## Desciption
+This file try to define how the associations will be handled in our application.
+Firstly, let see some user stories linked to associations. Then, how we can implement those functionalities :
+## User Stories
+- As a user interested in philosophy, I want to be able to find related associations so that I don't have to make too much effort.
+- As a user potentially interested in joining JDRPoly, I want to be able to get enough information on this association so that I know if it interests me or not.
+- As a curious user, I want to be able to discover new associations that I never thought of so that I can start a new activity.
+- As a user, I want to be able to follow an association so that I can easily see news and events related to it.
+- As a user in charge of communication in my association, I want to be able to share latest news so that people can see what's happening in my association.
+- As a user in charge of communication in my association, I want to be able to create events and certified them with my association so that other users can see that my event is from my association.
+## Implementation
+### Dataclass Association
+We already have a dataclass called Association. Currently, it has the following attributes : an "associationId", a "name" and a "description". We can add to this :
+- "relatedTags: Set<Tag>", the set of all tags related to the association
+- "moderators: Set<User>", the set of people that have the right to publish actuality, modify association description, add members and create events in the name of the association
+- TO DISCUSS : "members: Set<User>", list of all approved members of the association
+- "e-mail : String", a reference to the e-mail of the association if users want to contact them
+- "website: String? = "" ", a reference to the website of the association for those who have one
+- "instagram: String? = "" ", a reference to the instagram of the association for those who have one
+### AssociationsScreen
+In the hamburger menu, we can add a route to go to an AssociationsScreen where we can find everything related to associations. 
+The first thing a user see is the recent actuality of each associations she/he follows, like a feed (For simplicity, instead of Actuality, we can firstly just show all events created by the association). 
```

Comment:
```suggestion
The first thing a user sees is the recent news of each associations she/he follows, like a feed (For simplicity, instead of news, we can firstly just show all events created by the association). 
```
Again, I don't think our app is really about "news" in general. we should probably talk about events instead.